ding_nis.module

```diff
@@ -79,10 +75,13 @@ function _ding_nis_alter_image_fields() {
 
     // Getting field base settings.
     $base = field_read_field($field);
-    // Force cardinality to -1.
-    $base['cardinality'] = '-1';
-    // Update field base with unlimited cardinality.
-    field_update_field($base);
+    // Check if cardinality is correctly set.
+    if($base['cardinality'] != '-1') {

```

Comment:
please add space after if statement.
It should look like this
`if ($base['cardinality'] != '-1') {`